.storybook/components/CodeViewer/CodeViewer.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { v4 as uuid } from "uuid";
```

Comment:
I thought we're using nanoid now?

Reply:
I had a problem using nanoid... it seemed like it wasn't transpiled to es5, and Storybook would choke. I'm going to try again...

Reply:
Nope. `Object(...) is not a function`.  I could configure Storybook's webpack to transpile that specific package... or I can just use `uuid`.

Reply:
are you doing `import nanoid` or `import { nanoid }`?

Reply:
The former.  Like everyone else is...
https://github.com/acl-services/paprika/blob/master/packages/DataGrid/src/hooks/useGridEventHandler.js#L2

Is that wrong?

Reply:
that uses the previous major version. I'm reading some accounts that you might need to do the latte rnow

Reply:
Cool. Thanks!

Reply:
They do seem to be shipping untranspiled code. Wonder if this will break storybook in IE
https://unpkg.com/browse/nanoid@3.0.2/index.browser.js

Reply:
> Wonder if this will break storybook in IE
Who cares? 🤷‍♂

Reply:
Hmm right, anyone potentially on windows in Bangalore will have Edge right?

Reply:
...yeah, or CHROME! 😆

Reply:
lol... too tired 😪